core/trino-main/src/main/java/io/trino/sql/analyzer/ExpressionAnalyzer.java

```diff
@@ -3137,7 +3139,8 @@ private void coerceType(StackableAstVisitorContext<Context> context, Expression
             coerceType(expression, actualType, expectedType, message);
         }
 
-        private Type coerceToSingleType(StackableAstVisitorContext<Context> context, Node node, String message, Expression first, Expression second)
+        @FormatMethod
```

Comment:
Technically, doesn't belong to `Annotate semanticException factories as format methods` commit